indico/modules/events/papers/templates/_contributions.html

```diff
@@ -1,10 +1,11 @@
 {% macro render_paper_section(contribution, show_paper=True) %}
+    {% set can_access = (contribution.abstract and contribution.abstract.submitter == session.user) or contribution.is_user_associated(session.user) %}
```

Comment:
It's what we discussed during the meeting: If you submitted the abstract from which the contribution was created, you should be able to upload the paper for it even if you are not an author/speaker.